include/awkward/array/NumpyArray.h

```diff
@@ -174,7 +174,7 @@ namespace awkward {
                const std::vector<ssize_t>& strides,
                ssize_t byteoffset,
                ssize_t itemsize,
-               const std::string format,
+               const std::string& format,
```

Comment:
Good catch, static analyzer! I've been trying to pass anything that isn't just a bool/number by reference.